services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/appconfig/ApplicationConfigurationV2PublicationConfig.java

```diff
@@ -536,6 +541,20 @@ private DGCParameters buildDgcParameters(
         .setTestCertificateParameters(testCertificateParameters)
         .setExpirationThresholdInDays(distributionServiceConfig.getAppConfigParameters()
             .getDgcParameters().getExpirationThresholdInDays())
+        .setBlockListParameters(DGCBlocklistParameters.newBuilder()
+            .addAllBlockedUvciChunks(buildBlockedUvciChunks(distributionServiceConfig.getAppConfigParameters()
+                .getDgcParameters().getBlockListParameters().getBlockedUvciChunks()))
+            .build())
         .build();
   }
+
+  private List<DGCBlockedUVCIChunk> buildBlockedUvciChunks(
+      List<DgcBlockedUvciChunk> deserializedBlockedUvciChunks) {
+    return deserializedBlockedUvciChunks.stream().filter(dgcBlockedUvciChunk ->
+        TimeUtils.getNow().getEpochSecond() >= dgcBlockedUvciChunk.getValidFrom())
+        .map(deserializedBlockedUvciChunk -> DGCBlockedUVCIChunk.newBuilder()
+            .addAllIncides(deserializedBlockedUvciChunk.getIndices())
```

Comment:
I guess this will need to be adjusted for the typo once the ProtoBuf is updated.